### ISSUE TITLE
Can't seem to switch languages using api methods

### ISSUE THREAD

**Issue by ranilianaws** [aws-samples/amazon-sumerian-hosts#8]
When trying to use a different language than what was initialized I’m getting the following error in the console:

```
sequential_executor.js:108 Uncaught (in promise) UnexpectedParameter: Unexpected key 'Language' found in params
    at ParamValidator.fail (http://localhost:3000/static/js/1.chunk.js:397338:37)
    at ParamValidator.validateStructure (http://localhost:3000/static/js/1.chunk.js:397365:14)
    at ParamValidator.validateMember (http://localhost:3000/static/js/1.chunk.js:397374:21)
    at ParamValidator.validate (http://localhost:3000/static/js/1.chunk.js:397322:10)
    at Request.VALIDATE_PARAMETERS (http://localhost:3000/static/js/1.chunk.js:395478:42)
    at Request.callListeners (http://localhost:3000/static/js/1.chunk.js:400999:20)
    at callNextListener (http://localhost:3000/static/js/1.chunk.js:400986:12)
    at http://localhost:3000/static/js/1.chunk.js:395431:9
    at finish (http://localhost:3000/static/js/1.chunk.js:391253:7)
    at http://localhost:3000/static/js/1.chunk.js:391271:9
    at CognitoIdentityCredentials.get (http://localhost:3000/static/js/1.chunk.js:391847:7)
    at getAsyncCredentials (http://localhost:3000/static/js/1.chunk.js:391265:24)
    at Config.getCredentials (http://localhost:3000/static/js/1.chunk.js:391287:9)
    at Request.VALIDATE_CREDENTIALS (http://localhost:3000/static/js/1.chunk.js:395423:26)
    at Request.callListeners (http://localhost:3000/static/js/1.chunk.js:400994:18)
    at Request.emit (http://localhost:3000/static/js/1.chunk.js:400966:10)
    at Request.emit (http://localhost:3000/static/js/1.chunk.js:399565:14)
    at Request.transition (http://localhost:3000/static/js/1.chunk.js:398898:10)
    at AcceptorStateMachine.runTo (http://localhost:3000/static/js/1.chunk.js:405036:12)
    at Request.runTo (http://localhost:3000/static/js/1.chunk.js:399274:15)
    at http://localhost:3000/static/js/1.chunk.js:399673:12
    at new Promise (<anonymous>)
    at Request.promise (http://localhost:3000/static/js/1.chunk.js:399661:12)
    at TextToSpeechFeature._synthesizeSpeechmarks (webpack-internal:///333:4916:71)
    at TextToSpeechFeature._updateSpeech (webpack-internal:///333:4829:42)
    at eval (webpack-internal:///333:4785:18)
    at Array.forEach (<anonymous>)
    at TextToSpeechFeature._updateConfig (webpack-internal:///333:4772:41)
    at TextToSpeechFeature._getSpeech (webpack-internal:///333:5019:21)
    at TextToSpeechFeature.play (webpack-internal:///333:5241:19)
    at eval (webpack-internal:///333:6067:140)

```
The way I’m doing it is by passing in options as follows:
```

  host.TextToSpeechFeature.play(text, {
      Engine: this.languageMap[this.language].engine,
      Language: this.languageMap[this.language].languageName,
      VoiceId: this.languageMap[this.language].voiceId,
}) 
```

It seems like it's a bug. Unable to switch languages after initialization without hardcoding the entire config object.

**Comment by c-morten** [comment]
It looks like we're not validating the config objects for the play and resume methods like we do for the constructor, from the constructor we create the language code based on the language name. I will get this fixed, thanks for reporting this @ranilianaws.

**Comment by c-morten** [comment]
Fixed in v1.1.1
